package.json

```diff
@@ -1,12 +1,13 @@
 {
 	"scripts": {
 		"test": "xo && ava && npm run build",
-		"build": "rollup -c",
-		"watch": "run-p --continue-on-error --silent \"build -- --watch\"",
-		"watch:sourcemap": "cross-env SOURCEMAP=true npm run watch --silent",
+		"build": "webpack",
+		"watch": "webpack --watch",
+		"watch:sourcemap": "webpack --watch --devtool eval-cheap-module-source-map",
+		"reduce-size": "babel --out-dir . --no-comments extension/*.js",
```

Comment:
Ah true. That's weird how the options weren't working though. Maybe `babelrc: false` needs to be explicitely specified despite not existing anyway? Might be worth considering using [babili](https://github.com/babel/babili) too.